### PR TITLE
chore: test for footer EU

### PR DESCRIPTION
**Pull request by mahnoorgul-deriv**
Test for footer EU QATEST-1422 
Half test cases are covered for footer EU (responsive)
Changed Project structure to POM

### CODE REVIEW

**Review by markodonnell-deriv — COMMENTED** [review]
Looks good. Just some minor style comments. 

- We probably need to agree on use of ; If we're going to use it it ought to be everywhere. It's been left out on occasion in these two files. I think take it off for now.
- Also, spacing and alignment between it test ought to be consistent.
- Test descriptions need a slight improvement as per Chris G's comments in the #project_automated_functional_testing channel.
- Urls should be defined in the config file rather than hardcoded in the test.

File: cypress/e2e/smoke/footerEU.cy.js [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+import footerEuPage from '../../support/POM/commonEuPage';
+describe('QATEST-1422 Footer EU Responsive', () => {
+  beforeEach(() => {
+    cy.c_visitResponsive(Cypress.env('RegionEU'), 'small')
+  });
+  it('Footer logo and Social media icons verification for EU', () => {
+    footerEuPage.elements.footerLogo().should('be.visible')
+    footerEuPage.areSocialLinksVisible();
+  });
+  it('Footer Social Media Links Verification for EU', () => {
```
Comment (habib-deriv) [inline cypress/e2e/smoke/footerEU.cy.js:11]: Hi @mahnoorgul-deriv, you can use `forEach` for similar kind of links, something like this example

  it('should have correct Footer Social Media Links for EU', () => {
    const socialLinks = [
      { element: footerEuPage.elements.footerFaceBookLogo, url: footerEuPage.externalEUURls.facebookEU },
      // ... (repeat for other social links)
    ];

    socialLinks.forEach((link) => {
      footerEuPage.areSocialLinksCorrect(link.element, link.url);
    });
  });

File: cypress/e2e/smoke/footerEU.cy.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    footerEuPage.elements.aboutUsMenu().click();
+    footerEuPage.areAboutUsLinksVisible();
+    footerEuPage.elements.whoWeAreLink().click();
+    footerEuPage.elements.whoWeArePageText().should('be.visible');
+    footerEuPage.elements.whyChooseUsLink().click();
+    footerEuPage.elements.whyChooseUsPageText().should('be.visible');
+    footerEuPage.elements.partnershipProgrammesLink().click();
+    footerEuPage.elements.partnershipProgrammesPageText().should('be.visible');
+    cy.go(-1);
+    footerEuPage.elements.hamburgerMenu().click();
```
Comment (habib-deriv) [inline cypress/e2e/smoke/footerEU.cy.js:29]: footerEuPage.elements.hamburgerMenu().click() can be turned into a custom command and re used
```suggestion
    footerEuPage.elements.hamburgerMenu().click();
```

**Review by markodonnell-deriv — COMMENTED** [review]
Could you remove the fixture file and also the semi-colons. Thanks!

**Review by markodonnell-deriv — APPROVED** [review]

File: cypress/e2e/smoke/footerEU.cy.js (hunk 2 above)
Comment (mahnoorgul-deriv) [reply cypress/e2e/smoke/footerEU.cy.js]: done

**Pull request merged**